app/sidekiq/hca/base_submission_job.rb

```diff
@@ -41,5 +34,21 @@ def perform(user_identifier, encrypted_form, health_care_application_id, google_
 
       @health_care_application.set_result_on_success!(result)
     end
+
+    private
+
+    def handle_enrollment_system_validation_error(form, google_analytics_client_id)
+      StatsD.increment("#{HCA::Service::STATSD_KEY_PREFIX}.enrollment_system_validation_error")
```

Comment:
New StatsD metric. I know we have use a lot of different names for the `Veteran Enrollment System`, `VES`, `ES`, etc. Let me know if there is a better name for this metric.